cirq-google/cirq_google/serialization/serializable_gate_set.py

```diff
@@ -21,6 +21,9 @@
 from cirq_google.serialization import serializer, op_deserializer, op_serializer, arg_func_langs
 
 
+_GateOrFrozenCircuitTypes = Union[Type[cirq.Gate], Type[cirq.FrozenCircuit]]
```

Comment:
Optional: This is probably borderline, but I think inlining is still better here than defining an extra type. This helps in two ways: it avoids forcing someone reading the code to go to another module for the definition and it avoids using a private symbol from another module.

Up to you.

Reply:
Thanks for catching this - I agree reusing private symbol from another module breaks its privacy.
I changed that code to just define the type-alias locally - e74288381696865f0f1d2d7b0601284c4610552c.
[Personally I find it a bit easier to parse alias name than a type Union expression]